puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_plugin_matrix.yaml

```diff
@@ -60,7 +60,15 @@
           values:
             - fast
     execution-strategy:
-      combination-filter: "!(foreman_branch == 'develop' && ruby == '1.9.3')"
+      combination-filter: >
+        "!(foreman_branch == 'develop' && ruby == '1.9.3') ||
```

Comment:
This condition (line) will likely be true for say, Ruby 2.0.0 on MySQL, causing it to run every sub-job apart from Ruby 1.9.3 on develop, I think.